src/include/daos_srv/pool.h

```diff
@@ -65,6 +65,12 @@ struct ds_pool {
 	uint32_t		sp_stopping:1,
 				sp_fetch_hdls:1,
 				sp_need_discard:1;
+
+	/* pool_uuid + map version + leader term + rebuild generation define a
+	 * rebuild job.
+	 */
+	uint32_t		sp_rebuild_gen;
```

Comment:
It seem not clear enough. If rebuild generation is changed, then "leader term" also will be changed, right?

Reply:
No, leader term only change if the leader is switched.  But generation will change if the rebuild needs retry/reclaim due to failure.